.github/workflows/ci.yml

```
@@ -8,17 +8,16 @@ on: # yamllint disable-line rule:truthy
     branches: [main]
 
 concurrency:
-  group: "${{ github.head_ref || github.ref }}-${{ github.workflow }}"
+  group: '${{ github.head_ref || github.ref }}-${{ github.workflow }}'
   cancel-in-progress: true
 
 jobs:
-  lint-test:
-    name: Lint and Test
-    runs-on: ubuntu-22.04
+  lint:
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - name: Install Poetry
-        run: pipx install poetry==1.7.1
+        run: pipx install poetry==1.8.3 # keep in sync with pyproject.toml
```

Can you make sure the lock is generated using 1.8.4. I've had some issues when the lock was generated with a different version.
```suggestion
        run: pipx install poetry==1.8.4 # keep in sync with pyproject.toml
```